package.json

```diff
@@ -22,11 +22,12 @@
     "promote": "heroku pipelines:promote -r staging",
     "dump-schema": "babel-node ./scripts/dump-schema.js",
     "heroku-postbuild": "babel index.js config.js -s inline -d build && babel lib -s inline -d build/lib && babel schema -s inline -d build/schema",
-    "pre-commit": "lint-staged"
+    "precommit": "lint-staged"
```

Comment:
Eh, how do people feel about pre-commit hooks in general? We had one in `reaction-force` and took it out. Some people like to make lots of local commits and then push. Maybe `pre-push` is more appropriate?